### PR TITLE
complete search redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: content/2_Bronze/Intro_Complete.mdx [hunk 1]
```diff
@@ -31,44 +31,54 @@ $$
...
+	for (int& t : x) {
+		cin >> t;
+	}
+	for (int& t : y) {
+		cin >> t;
+	}
+	
+	int max_squared = 0;  // stores the current maximum
+	for (int i = 0; i < n; i++) {  // for each first point
+		for (int j = i+1; j < n; j++) {  // and each second point
```
Comment (ChuangSheep) [inline content/2_Bronze/Intro_Complete.mdx:57]:
```suggestion
		for (int j = i + 1; j < n; j++) {  // and each second point
```

File: content/2_Bronze/Intro_Complete.mdx [hunk 2]
```diff
@@ -159,39 +176,44 @@ variable.
...
 	double max_dist = 0;
-	for (int i = 0; i < N; i++) { // for each first point
-		for (int j = i+1; j < N; j++) { // for each second point
-			int dx = x[i]-x[j], dy = y[i]-y[j];
-			int square = dx*dx+dy*dy;
-			// if the square of the distance between the two points is greater than
-			// our current maximum, then update the maximum
-			max_dist = max(max_dist,sqrt(square));
+	for (int i = 0; i < n; i++) {
+		for (int j = i+1; j < n; j++) {
```
Comment (ChuangSheep) [inline content/2_Bronze/Intro_Complete.mdx:192]:
```suggestion
		for (int j = i + 1; j < n; j++) {
```

**Review by ChuangSheep — APPROVED** [review]
nice

**Comment by SansPapyrus683** [comment]
@ryanchou-dev

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**